go.mod

```diff
@@ -59,14 +59,12 @@ require (
 	github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
 	github.com/influxdata/influxdb1-client v0.0.0-20190118215656-f8cdb5d5f175
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/jonboulle/clockwork v0.1.0 // indirect
 	github.com/json-iterator/go v1.1.7 // indirect
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/kr/pty v1.1.8
 	github.com/krishicks/yaml-patch v0.0.10
-	github.com/lib/pq v0.0.0-20181016162627-9eb73efc1fcc
-	github.com/mattn/go-colorable v0.1.1
-	github.com/mattn/go-isatty v0.0.7
+	github.com/lib/pq v1.2.0
```

Comment:
I guess we could bump straight to v1.3.0 if we're bumping this, no big deal though.